doc/develop/application/index.rst

```diff
@@ -190,9 +190,15 @@ following example, ``app`` is a Zephyr freestanding application:
         └── src/
             └── main.c
 
+.. _zephyr-creating-app:
+
 Creating an Application
 ***********************
 
+In Zephyr, you can either use a reference workspace application or create your application by hand from scratch.
+
+.. _zephyr-creating-app-from-example:
+
 example-application
```

Comment:
I would rename this to make it clearer this is the first of the two options enumerated just before

```suggestion
Using a Reference Workspace Application (example-application)
```

Reply:
Even shorter without the "(example-application)" should be fine.

Reply:
True :)